301-multi-tier-loadbalancing/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Region1-location-name": {
+      "defaultValue": "Central US",
+      "allowedValues": [
+        "EastUS",
+        "EastUS",
+        "Central US",
+        "NorthCentralUS",
+        "SouthCentralUS",
+        "WestCentralUS",
+        "WestUS",
+        "WestUS2",
+        "USGovVirginia",
+        "USGovIowa",
+        "CanadaEast",
+        "CanadaCentral",
+        "BrazilSouth",
+        "North Europe",
+        "WestEurope",
+        "GermanyCentral",
+        "GermanyNortheast",
+        "UKWest",
+        "UKSouth",
+        "SoutheastAsia",
+        "EastAsia",
+        "AustraliaEast",
+        "AustraliaSoutheast",
+        "CentralIndia",
+        "WestIndia",
+        "SouthIndia",
+        "JapanEast",
+        "JapanWest",
+        "ChinaEast",
+        "ChinaNorth"
+      ],
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Azure Region valid name"
+      }
+    },
+    "Region2-location-name": {
+      "defaultvalue": "North Europe",
+      "allowedValues": [
+        "EastUS",
+        "EastUS2",
+        "Central US",
+        "NorthCentralUS",
+        "SouthCentralUS",
+        "WestCentralUS",
+        "WestUS",
+        "WestUS2",
+        "USGovVirginia",
+        "USGovIowa",
+        "CanadaEast",
+        "CanadaCentral",
+        "BrazilSouth",
+        "North Europe",
+        "WestEurope",
+        "GermanyCentral",
+        "GermanyNortheast",
+        "UKWest",
+        "UKSouth",
+        "SoutheastAsia",
+        "EastAsia",
+        "AustraliaEast",
+        "AustraliaSoutheast",
+        "CentralIndia",
+        "WestIndia",
+        "SouthIndia",
+        "JapanEast",
+        "JapanWest",
+        "ChinaEast",
+        "ChinaNorth"
+      ],
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Azure Region valid name"
+      }
+    },
+    "trafficManagerProfile_name": {
+      "defaultValue": "tmdemo",
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Traffice Manager profile name"
+      }
+    },
+    "adminuser": {
+      "defaultValue": "admin123",
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "adminuserPassword": {
+      "defaultValue": "TestUser@123",
+      "type": "securestring",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin Password"
+      }
+    },
+    "App-prefix": {
+      "defaultValue": "ab",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 3,
+      "metadata": {
+        "description": "Application prefix name, should be < 3 characters"
+      }
+    },
+
+    "imageSKU": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "allowedValues": [
+        "2008-R2-SP1",
+        "2012-Datacenter",
+        "2012-R2-Datacenter"
```

Comment:
No 2016? Not really required, but customers do tend to ask for it now that it's available